### PR TITLE
feat(error-label): custom styling for error message state

### PR DESCRIPTION
**Pull request by ZHRhodes**
Custom error message state to match designs. 

Design:
https://travelbank.invisionapp.com/share/6FBKY36WK#/screens/268269381

img_0259

ezgif-4-9cb6d6f485

### CODE REVIEW

**Review by Sh4rky — APPROVED** [review]
LGTM, check the indentation seems to be a little off

File: TextFieldEffects/TextFieldEffects/HoshiTextField.swift [hunk 1]
```diff
@@ -143,15 +145,18 @@ typealias VoidClosure = () -> Void
                        height: height)
         configurePlaceholderLabelFrame()
         configurePlaceholderFont()
+	configureErrorLabel()
```
Comment (xdansolo32) [inline TextFieldEffects/TextFieldEffects/HoshiTextField.swift:148]: is the spacing off in this and the next 2 addishes?

**Review by xdansolo32 — APPROVED** [review]
noice cleanup, small comment. also :vouch: gonzalo; the error text needs mroe left padding, and :notsureif: optical illusion but it seems like the error text is not in the same vertical posish as the normal header text.

not blockers doe~

File: TextFieldEffects/TextFieldEffects/HoshiTextField.swift (hunk 1 above)
Comment (ZHRhodes) [reply TextFieldEffects/TextFieldEffects/HoshiTextField.swift:148]: yeah I dont know what is happening with indentation. It was good on Xcode but wrong on github, then i fixed it for github file view but it still looks wrong is file comparison view. So if you click on the button to view the full file (not just changes) it's right..

**Comment by ZHRhodes** [comment]
@xdansolo32 vertical posish should be the same as the animated placeholder cause they use the same point as origin and have the same font text size

**Comment by ZHRhodes** [comment]
screen shot 2017-12-21 at 12 11 22 pm
@xdansolo32 pixel for pixel, the left padding is almost a perfect match for the mocks. but it looks a little off because the spacing between each part is a little different. Should be more spacing above the grey text in a perfect world

**Pull request merged**